charts/paperless/values.yaml

```diff
@@ -82,12 +82,24 @@ ports:
     protocol: TCP
 
 secret:
-  # -- Create a new secret containing the [secret key](https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security).
+  # -- Create a new secret containing the [secret values](https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security).
   create: true
-  # -- Use an existing secret to store the [secret key](https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security).
+  # -- Use an existing secret to store the [secret values](https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security). Please note: keys inside the existing secret must match the keys from below!
   existingSecret: ''
-  # -- [Secret key](https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security) used when not using an existing secret.
-  secretKey: ''
+  # -- Secret values used when not using an existing secret.
+  values:
+    # -- Secret key for session tokens.
+    - name: PAPERLESS_SECRET_KEY
+      value: '{{ randAlphaNum 42 | b64enc }}'
+    # -- Optional. Initial admin username.
+    # - name: PAPERLESS_ADMIN_USER
+    #   value: ''
+    # -- Optional. Initial admin password.
+    # - name: PAPERLESS_ADMIN_PASSWORD
+    #   value: ''
+    # -- Optional. Initial admin email.
+    # - name: PAPERLESS_ADMIN_MAIL
+    #   value: ''
```

Comment:
It's probably better to switch to a `dict` type object here to optimize `helm-docs` output. Also it's easier to check for missing/existing values...